values-nl/strings.xml

```diff
@@ -0,0 +1,132 @@
+    <string translatable="false" name="banner_ad_unit_id">ca-app-pub-3187897317987192/7514856660</string>
+    <string name="spanish_translations_credits" translatable="false">masterojitos</string>
+    <string name="german_translations_credits" translatable="false">yv35p00l</string>
+    <string name="french_translations_credits" translatable="false">Avenbeard</string>
+    <string name="russian_translations_credits" translatable="false">LionisIAm</string>
+    <string name="dutch_translations_credits" translatable="false">Tashla</string>
+
+    <string name="pokemesh_title_activity_maps">Kaart</string>
+    <string name="pokemesh_scan">Scannen</string>
+    <string name="pokemesh_login">Aanmelden</string>
+    <string name="pokemesh_settings">Instellingen</string>
+    <string name="pokemesh_search">Zoeken</string>
+    <string name="pokemesh_search_something">Iets zoeken…</string>
+    <string name="pokemesh_search_address">Adres</string>
+    <string name="pokemesh_search_address_hint">Een adres zoeken</string>
+    <string name="pokemesh_pokemon">Pokémon</string>
+    <string name="pokemesh_search_pokemons">Zoek een Pokémon</string>
+    <string name="pokemesh_stop_scan">Scan stoppen</string>
+    <string name="pokemesh_finding_pokemons">Nabije Pokémon zoeken…</string>
+    <string name="pokemesh_login_error">Aanmelden mislukt! Mogelijk zijn de Pokémon GO servers offline of niet bereikbaar!</string>
+    <string name="pokemesh_pokemesh_summary">Automatische nabije Pokémon detector. Door Pokémesh te gebruiken krijg je notificaties als een Pokémon bij je in de buurt is.</string>
+    <string name="pokemesh_last_seen">Laatst gezien: %s</string>
+    <string name="pokemesh_never">Nooit</string>
+    <string name="pokemesh_on_map">Op de kaart!</string>
+    <string name="pokemesh_gps_disabled">GPS is niet beschikbaar.</string>
+    <string name="pokemesh_my_position">Mijn positie</string>
+    <string name="pokemesh_no_permission">PokéMesh heeft geen toegang tot je positie!</string>
+    <string name="pokemesh_show_on_map">Op kaart weergeven</string>
+    <string name="pokemesh_show_on_map_summary">Geef deze Pokémon wel of niet weer op de kaart</string>
+    <string name="pokemesh_mesh_pokemon">Notificaties</string>
+    <string name="pokemesh_mesh_pokemon_summary">Geef een notificatie als deze Pokémon in de buurt is</string>
+    <string name="pokemesh_pokemesh_disabled">You need to enable PokéMesh settings before proceed!</string>
+    <string name="pokemesh_mesh_disabled">Notificaties uitgeschakeld</string>
+    <string name="pokemesh_mesh_enabled">Notitificaties ingeschakeld</string>
+    <string name="pokemesh_mesh_finding">Geen nabije Pokémon op het moment!</string>
+    <string name="pokemesh_meshed">Een %s is in de buurt!</string>
+    <string name="pokemesh_multi_meshed" formatted="false">Er zijn %s in de buurt van %s!</string> <!-- Questionable! Not sure what the last parameter is used for! -->
```

Comment:
%s are strings injected by the code. In this case %s will be replaced by the Pokémon name and count!